### PR TITLE
Feat/new default header [Delivers #114337573]

### PR DESCRIPTION
**Pull request by chiefGui**
@chute/frontend

### CODE REVIEW

File: src/components/live-displays/Header/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, {Component, PropTypes} from 'react'
 import './Header.scss'
 
+import ChuteLogo from '../../../images/chute_logo.svg'
```
Comment (jonathan-potter) [inline src/components/live-displays/Header/Header.jsx]: `../../..`...

Comment (jonathan-potter) [inline src/components/live-displays/Header/Header.jsx]: why not just add `images` to `aliases` in `webpack`?

File: src/components/live-displays/Header/Header.jsx [hunk 2]
```diff
@@ -20,12 +22,15 @@ export default class Header extends Component {
       segment2,
       segment2Color,
       backgroundColor,
-      fontSize
+      fontSize,
+      logoEnabled
     } = this.props
 
+    const isLogoEnabled = logoEnabled
```
Comment (jonathan-potter) [inline src/components/live-displays/Header/Header.jsx]: it seems pretty silly to create `isLogoEnabled` when `logoEnabled` is exactly as readable

if you REALLY want to rename `logoEnabled` to `isLogoEnabled` you can just do it in the destructuring:
```js
let {originalName: newName} = someObject
```

**Comment by jonathan-potter** [comment]
:+1: seems fine. I do think you should avoid relative paths generally, but definitely avoid them when you get too a `../../../../..` situation. No one knows where that is pointing.

**Comment by chiefGui** [comment]
Can you guys check this once again? @chute/frontend

**Comment by kevinrogerteng** [comment]
:+1:

**Pull request merged**